R/usethis-deprecated.R

```diff
@@ -0,0 +1,164 @@
+#' Deprecated Git functions
+#'
+#' @description
+#' `r lifecycle::badge("deprecated")`
+#'
+#' * `git_branch_default()` has been replaced by [git_default_branch()].
+#'
+#' @keywords internal
+#' @export
+git_branch_default <- function() {
+  lifecycle::deprecate_soft("2.1.0", "git_branch_default()", "git_default_branch()")
+  git_default_branch()
+}
+
+#' Deprecated tidyverse functions
+#'
+#' @description
+#' `r lifecycle::badge("deprecated")`
+#'
+#' * `use_tidy_labels()` has been replaced by [use_tidy_github_labels()].
+#' * `use_tidy_ci()` has been replaced by [use_tidy_github_actions()].
+#' * `use_tidy_eval()` is deprecated because there's no longer a need to
+#'    systematically import and re-export a large number of functions in order
+#'    to use tidy evaluation. Instead, use [use_import_from()] to tactically
+#'    import functions as you need them.
+#'
+#' @keywords internal
+#' @export
+use_tidy_labels <- function() {
```

Comment:
IMO you could remove these since they're mostly tidyverse internal.

Reply:
Yeah I had had the same thought. I'll go for it.